### DISCUSSION TITLE
Lineamientos proyecto final

### DISCUSSION THREAD

**Discussion by AlejandroBecerraAcevedo** [rramosp/ai4eng.v1#302] [00 - General]
Cordial saludo,

Queria saber si en el documento subido a github ¿hay algún problema si se crea un virtual environment? y tambien si ¿es obligatoria la extensión .ipynb o se puede hacer en .py?

**Comment by Felipe-RA** [comment]
Saludos @AlejandroBecerraAcevedo 

No hay problema en que incluyas un ``.venv``, un ``requirements.txt`` o similares. Pero su uso debe estar debidamente documentado en el *README*

Si requerimos encontrar **jupyter notebooks** (`.ipynb`) en los cuales se pueda ver un flujo de la ejecución del código organizado.

Cualquier otra duda, estamos pendientes.

**Reply by AlejandroBecerraAcevedo** [reply]
muchas gracias
